.gitlab-ci.yml

```diff
@@ -173,7 +173,6 @@ after_script:
   script:
     - cd _install_ci
     - find lib/coq/ -name '*.vo' -print0 > vofiles
-    - for regexp in 's/.vo//' 's:lib/coq/plugins:Coq:' 's:lib/coq/theories:Coq:' 's:/:.:g'; do sed -z -i "$regexp" vofiles; done
     - xargs -0 --arg-file=vofiles bin/coqchk -silent -o -m -coqlib lib/coq/
```

Comment:
Since there's no need for the sed step we may as well pipe instead of going through the `vofiles` file

Reply:
Ok, if it works, I'll adjust it.

Reply:
Done.